VideoApp/VideoApp/API/Passcode/Core/APIConfig.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Copyright (C) 2020 Twilio, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+struct APIConfig: Equatable {
```

Comment:
I think host probably is better on the API client as apposed to the request because normally host is just configured once based on environment or build variant and then the requests that the client executes are just concerned about path like `token`. So I think it's less duplication if client has host and normally an app is just using one host at a time. If we had to use multiple hosts at the same time it might make sense to create 2 instances of API client because normally there are other things that are a factor also like auth stuff.

You are correct that right now the only thing APIConfig buys us is it's easier to make the client more configurable in the future. When I make old API code use new API client I think I will need to make response handling configurable. I think I'll need to create a response handler for each domain because the shape of the error response is different. And I was thinking I would specify this response handler in the config but haven't thought a lot about it.

But yeah maybe jumped the gun here a little. I'm thinking leave it in and rethink as I make changes for old API code.

Reply:
Sounds good. I just wanted to get your rationale... and it was pretty much what I was thinking you'd say. I'm good with it and we can see how it shakes out with the next PR.